From 2fa4d2369c2d9dd583d035b6b344dcd139cfce72 Mon Sep 17 00:00:00 2001
From: Liwei Song <liwei.song@windriver.com>
Date: Mon, 24 Feb 2020 10:52:55 +0800
Subject: [PATCH] lable system-export as ostree_exec_t

Label system-export as ostree_exec_t to allow it manage files
in /etc

Upstream-Status: Pending

Signed-off-by: Liwei Song <liwei.song@windriver.com>
---
 policy/modules/admin/ostree.fc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/policy/modules/admin/ostree.fc b/policy/modules/admin/ostree.fc
index a595b5d8c..80cb2c6a4 100644
--- a/policy/modules/admin/ostree.fc
+++ b/policy/modules/admin/ostree.fc
@@ -1,3 +1,4 @@
 # Core definitions
 /usr/bin/ostree_reset.sh	--	gen_context(system_u:object_r:ostree_exec_t,s0)
 /usr/bin/ostree	--	gen_context(system_u:object_r:ostree_exec_t,s0)
+/usr/bin/system-export		--	gen_context(system_u:object_r:ostree_exec_t,s0)
-- 
2.17.1

